src/main/java/com/internet/shop/controllers/UsersController.java

```diff
@@ -0,0 +1,32 @@
+package com.internet.shop.controllers;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.models.User;
+import com.internet.shop.services.UserService;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class UsersController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private UserService userService = (UserService) injector.getInstance(UserService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        List<User> users = userService.getAll();
+        req.setAttribute("users", users);
+        req.getRequestDispatcher("/WEB-INF/views/user/all.jsp").forward(req, resp);
+    }
```

Comment:
let's move this method to a new controller: GetAllUsersController

Reply:
moved